src/content/en/fundamentals/primers/shadowdom/index.markdown

```diff
@@ -956,18 +956,42 @@ Chrome 53 ([status](https://www.chromestatus.com/features/4667415417847808)), Op
 To feature detect shadow DOM, check for the existence of `attachShadow`:
 
 {% highlight javascript %}
-function supportsShadowDOM() {
-  return !!HTMLElement.prototype.attachShadow;
+const supportsShadowDOMV1 = !!HTMLElement.prototype.attachShadow;
+{% endhighlight %}
+
+#### Polyfill {#polyfill}
+
+Until browser support is widely available, the [shadydom](https://github.com/webcomponents/shadydom) and [shadycss](https://github.com/webcomponents/shadycss) polyfills give you v1 feature. Shady DOM mimics the DOM scoping of Shadow DOM and shadycss polyfills CSS custom properties and the style scoping the native API provides.
+
+Install the polyfills:
+
+    bower install --save webcomponents/shadydom
+    bower install --save webcomponents/shadycss
+
+Use the polyfills:
+
+{% highlight javascript %}
+function loadScript(src) {
+ return new Promise(function(resolve, reject) {
```

Comment:
I'd remove the Promise wrapper, since you're not using the Promise that's being returned.